images/macos/templates/macOS-13.arm64.anka.pkr.hcl

```diff
@@ -49,6 +49,16 @@ variable "xcversion_auth_cookie" {
   default = ""
 }
 
+variable "xcode_install_storage" {
```

Comment:
nit: may be rename to `xcode_install_storage_url`?

Reply:
done